### PR TITLE
Update examples for the new structure system

### PR DESCRIPTION
**Pull request by cfanoulis**
### Description of the PR
The examples were using the old `Node`-based system. This PR rewrites them to use the new `Server`/`Client` system

### Semver Classification

- [x] This PR only includes documentation or non-code changes.
- [ ] This PR fixes a bug and does not change the (intended) API/interface.
- [ ] This PR adds methods or properties to the API/interface.
- [ ] This PR removes or renames methods or properties in the API/interface.

### CODE REVIEW

File: examples/concurrent.ts [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
 
 const sleep = promisify(setTimeout);
 const TIMES = 10000;
 
-const node = new Node('concurrent')
-	.on('error', (error, client) => console.error(`[IPC] Error from ${client.name}:`, error))
-	.on('socket.disconnect', client => console.error(`[IPC] Disconnected from ${client.name}`))
-	.on('socket.ready', async client => {
+const node = new Client('concurrent')
+	.on('error', (error, client) => console.error(`[IPC] Error from ${client!.name}:`, error))
```
Comment (kyranet) [inline examples/concurrent.ts]: Fixed this bug in 877e9e67e81299214807b6fe5062e80f90c4ac51

Can you rebase, please?

Comment (kyranet) [reply examples/concurrent.ts]: Same for the `interactive` and `world` examples.

**Comment by cfanoulis** [comment]
@kyranet please re-review

File: examples/concurrent.ts (hunk 1 above)
Comment (kyranet) [reply examples/concurrent.ts]: Please rename all instances of `client!` to `client`.

**Pull request merged**